### ISSUE TITLE
configure: WARNING: stdarg.h: present but cannot be compiled

### ISSUE THREAD

**Issue by cduston44** [flintlib/flint#1651]
when running configure, I was asked to report this warning.

```
checking stdarg.h presence... yes
configure: WARNING: stdarg.h: present but cannot be compiled
configure: WARNING: stdarg.h:     check for missing prerequisite headers?
configure: WARNING: stdarg.h: see the Autoconf documentation
configure: WARNING: stdarg.h:     section "Present But Cannot Be Compiled"
configure: WARNING: stdarg.h: proceeding with the compiler's result
configure: WARNING:     ## -------------------------------------------------------- ##
configure: WARNING:     ## Report this to https://github.com/flintlib/flint/issues/ ##
configure: WARNING:     ## -------------------------------------------------------- ##
checking for stdarg.h... no
configure: error: Could not find a mandatory header!
```
I guess the solution is to change configure.ac to check for prereq headers, but i'm afraid I don't have the confidence in my knowledge of the build process to make that change myself - just reporting what configure is telling me to report.

**Comment by cduston44** [comment]
I just didn't have libtool installed, sorry! I
